docs/updating/updating_to_3.3.md

```diff
@@ -206,3 +206,7 @@ Finish the update by running the following commands:
 php bin/console ibexa:graphql:generate-schema
 composer run post-install-cmd
 ```
+
+## Enable Commerce features
```

Comment:
As this belongs to 3.3.2 only, it should go above "Finish the update", to the "Updating to 3.3.2" section.